pkg/psmdb/mongo/mongo.go

```diff
@@ -477,6 +477,17 @@ func (m *ConfigMembers) SetVotes() {
 		if member.Hidden {
 			continue
 		}
+
+		if _, ok := member.Tags["nonVoting"]; ok {
+			// Non voting member is a regular ReplSet member with
+			// votes and priority equals to 0.
+
+			[]ConfigMember(*m)[i].Votes = 0
+			[]ConfigMember(*m)[i].Priority = 0
+
+			continue
+		}
+
```

Comment:
1) I wonder why 501-502 with `lastVoteIdx` lines needed
2) looks like 509-512 and `voting` counter not needed anymore - in this PR we allow to explicitly control the voting and non-voting members

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/723/commits/17c6fd26b7ec1cd6a44b604f53d60880a8ca4b4c

I kept the `votes` variable since by default there are no `non-voting` members and if a user creates a cluster with more than 7 mongod nodes, it'll exceed the maximum voting members limit.